pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -163,6 +163,9 @@ func NetlinkFlowToAntreaConnection(conn *conntrack.Flow) *flowexporter.Connectio
 		newConn.StopTime = time.Now()
 	}
 
+	if newConn.StartTime.IsZero() {
+		newConn.StartTime = time.Now()
+	}
```

Comment:
Every time we poll, we change the start time. This is not correct. We should set it the first time we see the connection. It is different from `StopTime`.

Reply:
Got it. Then there is possibility that startTime is later than stopTime?

Reply:
Yes if the connection is only polled once.
To resolve this, let's re-update the stopTime again after updating startTime when the connection is added to the map for the first time.
This should be done only for cases when there is zero startTime.